docs/running-on-kubernetes.md

```diff
@@ -932,15 +935,31 @@ specific to Spark on Kubernetes.
   <td>(none)</td>
   <td>
    Specify the local file that contains the driver [pod template](#pod-template). For example
-   <code>spark.kubernetes.driver.podTemplateFile=/path/to/driver-pod-template.yaml`</code>
+   <code>spark.kubernetes.driver.podTemplateFile=/path/to/driver-pod-template.yaml</code>
   </td>
 </tr>
 <tr>
   <td><code>spark.kubernetes.executor.podTemplateFile</code></td>
   <td>(none)</td>
   <td>
    Specify the local file that contains the executor [pod template](#pod-template). For example
-   <code>spark.kubernetes.executor.podTemplateFile=/path/to/executor-pod-template.yaml`</code>
+   <code>spark.kubernetes.executor.podTemplateFile=/path/to/executor-pod-template.yaml</code>
+  </td>
+</tr>
+<tr>
+  <td><code>spark.kubernetes.driver.podTemplateContainerName</code></td>
```

Comment:
Could you move this to line 941 after `spark.kubernetes.driver.podTemplateFile` config? We had better group together `spark.kubernetes.driver.*` configs and `spark.kubernetes.executor.*` configs.

Reply:
Thanks, done. I'll squash the commits before merging.